WordPress/Classes/ViewRelated/Domains/Domain registration/RegisterDomainSuggestions/RegisterDomainSuggestionsViewController.swift

```diff
@@ -111,6 +111,14 @@ class RegisterDomainSuggestionsViewController: UIViewController {
         hideButton()
         setupTransferFooterView()
     }
+   
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        WPAnalytics.track(
+            .domainsDashboardDomainsSearchShown,
+            properties: [WPAppAnalyticsKeySource: "all_domains"]
+        )
+    }
```

Comment:
[Suggestion] `viewDidAppear` can be called multiple times after a screen is loaded. For example, when pushing to a new screen then popping back to previous screen. What do you say we move this logic to `viewDidLoad`?

Reply:
I can do that. I think the shown events should trigger whenever the screen is shown but I'm not sure what we've been following in our codebase.